rest_framework_reactive/routing.py

```diff
@@ -0,0 +1,7 @@
+from .consumers import ObserversConsumer
+
+
+channel_routing = [
+    # TODO: Should the 'ws' prefix be made configurable?
```

Comment:
Yes, it would be better. You can set `/ws/` as default.

Reply:
Well, with channels you can always override this by just using the `ObserversConsumer` in a custom routing scheme instead of importing `rest_framework_reactive.routing`, similarly to how you can configure urls. So perhaps adding an explicit configuration setting would be strange?

Reply:
Ok, i agree. Maybe you can rename `channel_routing` to `default_channel_routing` or something similar.